ci/doc/how-to-setup-taos-ci-server.md

```diff
@@ -31,13 +34,12 @@ $ firefox http://localhost/index.php
 Note. If you have a firewall in your network, please make sure that ports for CI server are opened and can accept requests.
 
 ## How to set-up domain name address
-If you want to use your own domain name address instead of IP address, we recommend that you try to get a host name
-free of charge at https://freedns.afraid.org.
+If you want to use your own domain name address instead of IP address for effective maintenance, we recommend that you try to get a host name free of charge at https://freedns.afraid.org.
 
 ```bash
 $ sudo vi /etc/apache2/sites-enabled/000-default.conf 
 <VirtualHost *:80>
-        # You can get a free dns such as {your_host}.mooo.com free of charge at http://freedns.afraid.org.
+        # You can get five host names such as {your_host}.mooo.com free of charge at http://freedns.afraid.org.
```

Comment:
What is the five host names?

Reply:
As I mentioned, user can get  host names until 5 free of charge.